apps/backend/src/constants.ts

```diff
@@ -1 +1,33 @@
+import {
+  LeaderboardEntryType,
+  OAuthMapping
+} from "@types";
+
 export const FRONTEND_URL = process.env.FRONTEND_URL || "http://localhost:3000";
+
+export const OAUTH_APP_MAPPING: Record<string, OAuthMapping> = {
+  "strava": {
+    client_side_fetching: true,
+    token_url: "https://www.strava.com/api/v3/oauth/token",
+    redirect_uri: "http://connections.cursive.team/oauth/exchange_token&approval_prompt=force&scope=read",
+    id: process.env.NEXT_PUBLIC_OAUTH_STRAVA_CLIENT_ID || "",
+    secret: process.env.NEXT_PUBLIC_OAUTH_STRAVA_CLIENT_SECRET || "",
+    data_options: [
+      {
+        type: LeaderboardEntryType.STRAVA_PREVIOUS_MONTH_RUN_DISTANCE,
+      }
+    ],
+  },
+  "github": {
+    client_side_fetching: false,
+    token_url: "https://github.com/login/oauth/access_token",
+    redirect_uri: "http://connections.cursive.team/oauth/exchange_token&approval_prompt=force&scope=read",
```

Comment:
@AndrewCLu for testing github you also need to change the domain to localhost:3000, sorry for not mentioning earlier.